python/ray/rllib/train.py

```diff
@@ -102,10 +113,19 @@ def run(args, parser):
         if not exp.get("env") and not exp.get("config", {}).get("env"):
             parser.error("the following arguments are required: --env")
 
-    ray.init(
-        redis_address=args.redis_address,
-        num_cpus=args.ray_num_cpus,
-        num_gpus=args.ray_num_gpus)
+    if args.ray_num_local_schedulers:
+        ray.worker._init(
```

Comment:
I’d prefer to deprecate this way of starting multiple raylets and instead just use @richardliaw’s ‘cluster_utils’.